src/ui/components/Button.qml

```diff
@@ -49,4 +49,8 @@ QQC.Button {
 
     property alias tooltip: tt.text;
     ToolTip { id: tt; visible: text.length > 0 && root.hovered; }
+    
+    // Enable enter to click
+    Keys.onReturnPressed: root.onClicked();
+    Keys.onEnterPressed: root.onClicked();

```

Comment:
does this work? I think it should be `root.clicked()`

Reply:
Okay now I'm pretty convinced that you never sleep :). It works since it calls the handler function defined when the UI element is initialised, but changing it does look more elegant. There's a bug with the ComboBox enter key so I'll try fixing that and changing this as well.

Reply:
but this is a signal named `clicked`, the handler is `onClicked`. In QML you should always trigger signals because it will notify all listeners this way, instead of calling the handler. I didn't check this now, but if `root.clicked()` works, it should be `root.clicked()`.